content/src/outline.rs

```diff
@@ -120,6 +120,17 @@ impl Outline {
         &self.contours
     }
 
+    #[inline]
+    pub fn clear(&mut self) {
+        self.contours.clear();
+        self.bounds = RectF::default();
+    }
+    
+    #[inline]
+    pub fn len(&self) -> usize {
+        self.contours.len()
+    }
+    
```

Comment:
also used by vector

Reply:
these are used to improve performance (and they do a good job at it).
Turns out copying the known length into a newly allocated buffer, and then reusing the old one for the next path, is a lot faster than starting with an empty buffer each time and transferring ownership.